lib/TWCManager/Control/MQTTControl.py

```diff
@@ -95,6 +95,8 @@ def mqttMessage(self, client, userdata, message):
             if len(plsplit) == 2:
                 self.master.setChargeNowAmps(int(plsplit[0]))
                 self.master.setChargeNowTimeEnd(int(plsplit[1]))
+                self.master.getModuleByName("Policy").applyPolicyImmediately()
+                self.master.queue_background_task({"cmd": "saveSettings"})
```

Comment:
Not technically required by this change, but a difference between the MQTT and HTTP paths.